x11-proto/kbproto/kbproto-1.0.7.recipe

```diff
@@ -3,39 +3,40 @@ DESCRIPTION="Headers for the X11 KB protocol."
 HOMEPAGE="http://xorg.freedesktop.org/releases/individual/proto/"
 COPYRIGHT="1993 by Silicon Graphics Computer Systems, Inc."
 LICENSE="MIT"
-REVISION="2"
-SOURCE_URI="http://xorg.freedesktop.org/releases/individual/proto/kbproto-1.0.7.tar.gz"
+REVISION="3"
+SOURCE_URI="https://xorg.freedesktop.org/releases/individual/proto/kbproto-$portVersion.tar.gz"
 CHECKSUM_SHA256="828cb275b91268b1a3ea950d5c0c5eb076c678fdf005d517411f89cc8c3bb416"
 
-ARCHITECTURES="x86_gcc2 x86 x86_64"
-SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+ARCHITECTURES="any"
 
 PROVIDES="
 	kbproto$secondaryArchSuffix = $portVersion
```

Comment:
```diff
-	kbproto$secondaryArchSuffix = $portVersion
+	kbproto = $portVersion
```

Reply:
Since we're dropping snd arch for this recipe also, could you check if there's any package requiring the snd arch version?

Reply:
IIRC the packages requiring the secondary arch version also are in the ones listed for util-macros